esp/esp/program/forms.py

```diff
@@ -62,23 +62,46 @@ class ProgramCreationForm(BetterModelForm):
+                                                        'Will you have extra costs (shirts or lunch)?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Optional Fees', 'Accounting', 'Financial Aid Application', 'Easily Approve Financial Aid Requests'])],
+                                                        'If you will charge for admission or other costs, will you accept payment by credit card?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Credit Card Payment Module (Stripe)', 'Credit Card View Module'])],
+                                                        'Do you want a pre-program quiz for teachers?': [x.id for x in ProgramModule.objects.filter(admin_title='Teacher Logistics Quiz')],
+                                                        'Will you have any additional non-survey forms that teachers should fill out?': [x.id for x in ProgramModule.objects.filter(admin_title='Teacher Custom Form')],
+                                                        'Will you have any (non-survey) forms that students should fill out?': [x.id for x in ProgramModule.objects.filter(admin_title='Student Custom Form')],
+                                                        'Will you have more than one lunch period (per day)?': [x.id for x in ProgramModule.objects.filter(admin_title='Student Lunch Period Selection')],
+                                                        'Would you be willing to solicit donations for LU?': [x.id for x in ProgramModule.objects.filter(admin_title='Donation Module')],
+                                                        'Do you plan to have teacher training or interviews?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Training and Interview Signups', 'Manage Teacher Training and Interviews'])],
+                                                        mark_safe('Will you use lottery admission (as opposed to first come, first served) for <b>classes</b>?'): [x.id for x in ProgramModule.objects.filter(admin_title__in=['Two-Phase Student Registration', 'Lottery Frontend'])],
+                                                        mark_safe('Will you use lottery registration (as opposed to first come, first served) to the <b>program</b>?'): [x.id for x in ProgramModule.objects.filter(admin_title__in=['Student Registration Phase Zero', 'Manage Student Registration Phase Zero'])],
+                                                        'Do students have to apply to individual classes?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Application Review for Admin', 'Admin Admissions Dashboard'])],
+                                                        'If yes, can teachers admit them (as opposed to just admins)?': [x.id for x in ProgramModule.objects.filter(admin_title__in=['Teacher Admissions Dashboard', 'Application Reviews for Teachers', 'Application Review for Admin', 'Admin Admissions Dashboard'])]
```

Comment:
For me, these questions showed up in a completely jumbled order on my site, which was disorienting and made the "If yes..." very confusing

Reply:
Oops, I guess the order on the ordered dict is undone by the fact that the internal dictionary is unordered. I should initialize it from a zipped list or something. Thanks for the catch

Reply:
Can't you just remove the curly braces?

Reply:
try after commit 64f6410